src/tbb/cache_aligned_allocator.cpp

```diff
@@ -19,15 +19,21 @@
 #include "tbb/tbb_allocator.h"
 #include "tbb/tbb_exception.h"
 #include "tbb_misc.h"
+#include "tbb_environment.h"
 #include "dynamic_link.h"
 #include <cstdlib>
+#include <stdlib.h> // posix_memalign
 
 #if _WIN32||_WIN64
 #include "tbb/machine/windows_api.h"
 #else
 #include <dlfcn.h>
 #endif /* _WIN32||_WIN64 */
 
+#if !_WIN32 && !_WIN64
+#define __TBB_USE_POSIX_MEMALIGN
+#endif
```

Comment:
What about [`_aligned_malloc`](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/aligned-malloc?view=vs-2019) on Windows?

Reply:
I can add it, although I don't use Windows and can't test.